src/Libraries/Watch3DNodeModels/Watch3D.cs

```diff
@@ -171,34 +171,10 @@ public void SetWatchSize(double w, double h)
 
         public override IEnumerable<AssociativeNode> BuildOutputAst(List<AssociativeNode> inputAstNodes)
         {
-            if (IsPartiallyApplied)
```

Comment:
If partially-applied Watch3d would output a function.

Here in this PR's implemenation. 

> 1.A few questions and 2. I think this needs tests for the new methods / interactions.

OK will add the tests :)

Reply:
I am not sure what IsPartiallyApplied mean? (I am still quite ignorant of many of Dynamo's inner working details) Does this mean the node will output a Function when the input is a Function? If so, this behavior is still maintined in this PR

![image](https://user-images.githubusercontent.com/9465346/232781584-e570a225-a764-47e7-8a53-a6118c12e32f.png)

Reply:
kind of, partially applied is a functional programming term - it means that a function has been created with some input parameters set and others still free.

Can you please confirm what happened before and after your changes if you actually invoke the function on some input, you can use `map` node - I don't think this is so important but honestly I have no idea what the behavior was, so let's see...

My feeling is that it used to just return the input object, lets see if it still does that?

Reply:
your screenshot is not what I mean - 
we want the output of the Watch3D node to be a function - or rather, I am trying to understand if that used to work and now it is broken - which I am guessing is the case because of the code you have removed.

Reply:
@mjkkirschner You are right, in the current master branch the Watch3D node will output a function (rather than "null) if the only-input port is free.

So looks like this PR will break any existing graph that relies on this function-output behavior, even thought I doubt if anyone would use Watch3D in such a way.